docs/reference/commandline/dockerd.md

```diff
@@ -1231,10 +1231,12 @@ The `--metrics-addr` option takes a tcp address to serve the metrics API.
 This feature is still experimental, therefore, the daemon must be running in experimental
 mode for this feature to work.
 
-To serve the metrics API on localhost:1337 you would specify `--metrics-addr 127.0.0.1:1337`
-allowing you to make requests on the API at `127.0.0.1:1337/metrics` to receive metrics in the
+To serve the metrics API on localhost:9323 you would specify `--metrics-addr 127.0.0.1:9323`
```

Comment:
Looks like this was missed when this was added originally; can you put back-ticks around `localhost:9323` ?

```suggestion
To serve the metrics API on `localhost:9323` you would specify `--metrics-addr 127.0.0.1:9323`
```